ispo-working-group-charter.md

```diff
@@ -1,41 +1,45 @@
 ## Summary
 
-We believe that most training and teaching content can be shared across companies when those institutions share the same relevant points of context.
-While not every company is exactly the same, not every company is wholly different, either.
-For each challenge in InnerSource, we believe that there are a small number of ways of common ways to address it that should work for most companies.
-Our goal is to document these sets of solutions to challenges, make it easy for ISPOs to find which apply to them, and then to produce sharable, open source implementations of these solutions.
-These implementations will likely be documents, videos, or code, each of which is generic enough to be used across companies,
-yet specific enough that people in the company can leverage them to solve their problems without needing notable amounts of company-specific work.
+The majority of training and teaching content can be shared across companies when the contexts are similar.  
+
+While not every company in InnerSource is the same, not every company is different. For each challenge in InnerSource, a few standard solutions may work for most.  
+
+We document standard solutions to challenges, make them discoverable, and provide open source implementations of these solutions.  
+
+Documents, videos, or code will likely be implemented, each generic yet specific enough to be used across companies without requiring significant company-specific work to solve problems.  
 
 ## Examples
 
-As an example, to the challenge of `How do I sell InnerSource to Senior Leadership?`,
-this working group could ensure that there are video clips explaining various benefits of InnerSource from a Senior Leader perspective.
-There would be different clips that highlight different benefits of InnerSource,
-and the ISPO could select those videos that highlight the benefits that their Senior Leaders care about.
+The working group could introduce video clips to explain InnerSource's various benefits from a Senior Leader's perspective, for example, to answer the question:  
+
+*How do I sell InnerSource to Senior Leadership?*  
+
+IPSOs could curate InnerSource videos that benefit their Senior Leaders.  
 
-As another example, to the challenge of `How do I show InnerSource KPIs?`,
-this working group could ensure that there is a software project that will produce charts of InnerSource activity from sources like _GitHub_ and Jira.
-The project could be configured in a (finite) number of ways, depending on varying setup of _GitHub_ and Jira that affects tha report views.
+Taking the challenge:
 
-These two examples follow the same pattern - providing directly-usable materials that help ISPOs to advance their mission in their company.
+*How do I present InnerSource KPIs?*  
+
+Our group could create a software project to generate InnerSource activity charts from GitHub and Jira. Depending on GitHub and Jira report configurations, a project could be preconfigured in limited ways.  
+
```

Comment:
Are we thinking of an open source Jira plugin?

A prescriptive configuration of issues then JQL queries?

[Easy BI](https://marketplace.atlassian.com/apps/1211051/eazybi-reports-and-charts-for-jira?tab=pricing&hosting=cloud) is a robust commercial plugin we could recommend for those that can afford it.

Reply:
I would suggest to avoid talking about a particular tool in a document such as the charter, because tools, even taken as examples, tend to be divisive and get old faster than such a document which is meant to last a very long time (as @jeffabailey comment shows). 
I would rather say something like: 

How to I present Inner Source Kpis ?
This working group could ensure that there is a software project that will produce charts of InnerSource activity from 
sources commonly used in open development environments such as source code repository, issue tracker or any other collaborative project tool. The project could be configured in a (finite) number of ways, depending on varying setup of the selected tools that affects the report views.

Reply:
I had the same thought. A proposal for a report layout may be useful.

Reply:
Gave a commit suggestion for this.